### PR TITLE
add tests for S3 cache layer

### PR DESCRIPTION
**Pull request by dduugg**

### CODE REVIEW

File: server/db/s3Cache.js [hunk 1]
```diff
@@ -6,7 +7,7 @@ var s3 = require("./s3");
  * @default
  * @type {number}
  */
-var CACHE_LIFETIME_MS = Math.POSITIVE_INFINITY;
+var CACHE_LIFETIME_MS = Number.POSITIVE_INFINITY;
```
Comment (zindlerb) [inline server/db/s3Cache.js]: :smirk:

File: test/server/db/s3CacheSpec.js [hunk 2]
```diff
@@ -0,0 +1,302 @@
...
+
+var MOCK_CALLBACK = function(error, data) {
+  return error ? "error" : "data";
+};
+
+var MOCK_DATA = "data data";
+var MOCK_DATE = _.now();
+var MOCK_ERROR = new Error("mock error");
+var MOCK_SCHEMA_NAME = "mock-schema";
+var MOCK_SIZE = 1337;
```
Comment (zindlerb) [inline test/server/db/s3CacheSpec.js]: :neckbeard:

**Comment by zindlerb** [comment]
:+1:

**Comment by cmperez** [comment]
:trophy:

**Pull request merged**